overlord/state/notices_test.go

```diff
@@ -604,6 +735,50 @@ func (s *noticesSuite) TestWaitNoticesLongPoll(c *C) {
 	}
 }
 
+func (s *noticesSuite) TestWaitNoticesBeforeFilter(c *C) {
```

Comment:
don't we need test where this actually returns something has well? all the cases here seem to be len(notices) == 0 ?